<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="DataLoader_tab">
          <attribute name="title">
           <string>DataLoader</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout" rowstretch="1,3,1,2,1" columnstretch="1,1">
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>35</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="dl_pb_clear">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Clear loaded data</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="dl_pb_save_pickle">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Pickle-save data</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="dl_pb_summary">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Summary</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="dl_pb_save_repo">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save to Repository</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QTextBrowser" name="dl_tb_log_console"/>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,2,1,0" columnstretch="1,0,1">
               <property name="leftMargin">
                <number>50</number>
               </property>
               <property name="rightMargin">
                <number>50</number>
               </property>
               <property name="verticalSpacing">
                <number>5</number>
               </property>
               <item row="0" column="0" colspan="3">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="dl_pb_load_pickle">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Load Saved DT</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="3">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="dl_pb_load_data">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Load Data</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabBarClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>change_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dl_pb_load_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dl_pb_load_pickle</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_pickle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>change_size(int)</slot>
  <slot>load_data()</slot>
  <slot>load_pickle()</slot>
 </slots>
</ui>
